SSD/ssdDriver.cpp

```diff
@@ -1,14 +1,42 @@
 #include "device.h"
+#include <iostream>
+#include <string>
+#include <vector>
 
-class SSDDriver : public Device{
+using std::string;
+using std::vector;
+
+class SSDDriver : public Device {
 public:
+
+  void run(int argc, char *argv[]) {
+    vector<string> params = parseParams(argc, argv);
+
+    string command = params[1];
+    int lba = std::stoi(params[2]);
+    unsigned int value = std::stoul(params[3]);
```

Comment:
현석님 자료형 변경 필요해 보입니다.

```suggestion
    unsigned long value = std::stoul(params[3]);
```